src/blocks/sound.rs

```diff
@@ -724,7 +724,7 @@ impl Sound {
             self.text.set_text(if self.bar {
                 format_percent_bar(volume as f32)
             } else {
-                text.to_string()
+                text
```

Comment:
What lint was this one? Surprised I didn't see it before as I thought I had got all the trivial ones like this already.

Reply:
this was also a redundant clone